server/src/main/java/org/candlepin/service/impl/DefaultEntitlementCertServiceAdapter.java

```diff
@@ -323,6 +324,9 @@ private String getContentPrefix(Entitlement ent, boolean useContentPrefix)
         String contentPrefix, Map<String, EnvironmentContent> promotedContent) {
         Set<X509ExtensionWrapper> result = v3extensionUtil.getExtensions(ent,
             contentPrefix, promotedContent);
+        X509ExtensionWrapper typeExtension = new X509ExtensionWrapper(OIDUtil.REDHAT_OID + "." +
```

Comment:
@barnabycourt do we want to take the opportunity in this class to start baking in the org key into the certificate DN?